api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java

```diff
@@ -14,12 +14,18 @@
 import org.pmiops.workbench.model.Degree;
 import org.pmiops.workbench.model.DisseminateResearchEnum;
 import org.pmiops.workbench.model.Domain;
+import org.pmiops.workbench.model.Education;
```

Comment:
The newly added mappings appear to duplicate those defined here: https://github.com/all-of-us/workbench/blob/bb8aec8d448abeece258d3b53d0d925ede1d5473/api/src/main/java/org/pmiops/workbench/db/model/DemographicSurveyEnum.java

Reply:
This doesn't seem to have been addressed

Reply:
arghhhh